src/main/java/frc/robot/commands/StartBallStorage.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.command.InstantCommand;
+import frc.robot.subsystems.Storage;
+
+public class StartBallStorage extends InstantCommand {
+
+  public StartBallStorage() {}
+
+  public void execute() {
+    Storage.getInstance().storeBall();
```

Comment:
Storage.getInstance() could be set to an instance variable

Reply:
Are we using commands now? If so, we should be using the [new command API](https://docs.wpilib.org/en/latest/docs/software/commandbased/command-based-changes.html). I already added the vendordep for the new framework, so you should be able to just use it.

Reply:
This will be refactored when we implement the state machine